### ISSUE TITLE
Squad Details

### ISSUE THREAD

**Issue by NSmonish** [IEEE-VIT/Howzat#10]
In the Home page in the list of participating nations when clicked on a country’s name/flag, it should lead to a page where it will contain squad details of the respective nation.

**Comment by nayeem-rafsan** [comment]
would like to work on this

**Comment by rafu01** [comment]
I would like to work on this

**Comment by weberankit** [comment]
@NSmonish I would like to contribute to this . I know ReactJS, js CSS, and HTML. could you plz tell me after clicking on the country name what you want to show and plz assign it to me

**Comment by NSmonish** [comment]
> I would like to work on this

Yes you can start working on the issue and create a PR once complete.

**Comment by NSmonish** [comment]
> @NSmonish I would like to contribute to this . I know ReactJS, js CSS, and HTML. could you plz tell me after clicking on the country name what you want to show and plz assign it to me

When clicked on the country name/flag, it should take you to a separate page where details of the respective nation's squad is to be present.

**Comment by rafu01** [comment]
I've started working on it. Will open PR soon
